cmd/lifecycle/analyzer.go

```diff
@@ -15,55 +15,54 @@ import (
 	"github.com/buildpacks/lifecycle/auth"
 	"github.com/buildpacks/lifecycle/buildpack"
 	"github.com/buildpacks/lifecycle/cmd"
+	"github.com/buildpacks/lifecycle/image"
 	"github.com/buildpacks/lifecycle/platform"
 	"github.com/buildpacks/lifecycle/priv"
 )
 
 type analyzeCmd struct {
 	//flags: inputs
 	analyzeArgs
-	uid, gid int
+	stackPath string
+	uid, gid  int
 
 	//flags: paths to write data
 	analyzedPath string
 }
 
 type analyzeArgs struct {
-	imageName     string
-	layersDir     string
-	orderPath     string //nolint - Platform API >= 0.7
-	previousImage string //nolint - Platform API >= 0.7
-	runImageRef   string //nolint - Platform API >= 0.7
-	stackPath     string //nolint - Platform API >= 0.7
-	useDaemon     bool
-
-	platform06     analyzeArgsPlatform06
-	additionalTags cmd.StringSlice        //nolint Platform API >= 0.7
+	cacheImageRef    string
+	layersDir        string
+	outputImageRef   string
```

Comment:
Renamed from `imageName` ...now that analyzer can care about both output & previous images, it was too confusing.